.github/workflows/protectSecurityRelevantCode.yml

```
@@ -0,0 +1,172 @@
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4.1.7
+        with:
+          fetch-depth: 0 ##### Fetch all history for all branches
+
+      - name: Check Git Diff for protected files
+        id: check_protected_files
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:8:28: Double quote to prevent globbing and word splitting

(shellcheck)